data/interfaces/default/settings.html

```diff
@@ -813,6 +813,16 @@ <h3>Server Monitoring</h3>
                                         <input type="hidden" class="form-control" id="pms_update_distro" name="pms_update_distro">
                                     </div>
                                 </div>
+                                <div class="row">
+                                    <div class="col-md-2">
+                                        <label for="pms_update_notify_interval">Notify Interval</label>
```

Comment:
Rename all `pms_update_notify_interval` to `pms_update_check_interval`. Rename setting label `Update Check Interval`.